README.md

```diff
@@ -22,22 +22,23 @@ Add the assembly and new target to NLog.config:
 
 ```xml
 <?xml version="1.0" encoding="utf-8" ?>
-<nlog   xmlns="http://www.nlog-project.org/schemas/NLog.xsd"
-        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-        autoReload="true"
-        throwExceptions="false">
+<nlog xmlns="http://www.nlog-project.org/schemas/NLog.xsd"
+      xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+      autoReload="true"
+      throwExceptions="false">
 
-      <!-- extensions is not needed in NLog 4+ -->
-      <extensions>
+    <!-- extensions is not needed in NLog 4+ -->
+    <extensions>
         <add assembly="NLog.Etw" />
-      </extensions>
+    </extensions>
 
-      <targets async="true">
-        <target xsi:type="ExtendedEventTracing"
```

Comment:
isn't better to keep both examples?

Reply:
No one wants to use the other targets. They force you to use the same ETW-ProviderName.

The idea is that you should have a unique name for each application.

Reply:
Actually curious why no one have implemented `NLogEtwExtendedTarget`-constructor that takes EventSource as parameter. Then one could atleast use it from code.

Reply:
OK, so maybe drop that target in the next major version?

Reply:
Will try and modify it so one can provide a custom EventSource, that implements an interface that takes NLog.LogEventInfo and has an EventSource-property-getter, so the target can query if enabled.

Reply:
That's for the future, isn't? I can merge and release this one?

Reply:
I have a PR ready now. When this has been merged. But now the AppVeyor-queue is long because of your double-build-commits.

Reply:
But yes that idea is for the "future".